pkg/listeners/listeners.go

```diff
@@ -78,77 +84,115 @@ func ProcessAlertHandler(w http.ResponseWriter, r *http.Request) {
 	// Retrieve the alert search results
 	var alert splunk.Webhook
 
-	err := helpers.DecodeJSONRequestBody(w, r, &alert)
+	client, err := jira.DefaultClient()
+	if err != nil {
+		log.Printf("failed creating Jira client: %s\n", err.Error())
+		set500Response(w)
+		// Should this panic?  Return?
+		// How do we notify ourselves when Jira client failures are occurring?
+		// Alert on some metric?
+		return
+	}
+
+	err = helpers.DecodeJSONRequestBody(w, r, &alert)
 	if err != nil {
 		var mr *helpers.MalformedRequest
 		if errors.As(err, &mr) {
+			log.Printf("received malformed request: %s\n", mr.Msg)
+			// This is a client error, so we return the status code and message
 			http.Error(w, mr.Msg, mr.Status)
 		} else {
-			log.Println(err.Error())
-			http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
+			log.Printf("failed decoding JSON request body: %s\n", err.Error())
+			set500Response(w)
 		}
 		return
 	}
 
-	log.Println("Received alert from Splunk:", alert.Sid)
+	log.Println("received alert from Splunk:", alert.Sid)
 
-	// searchResults, err := splunk.RetrieveSearchFromAlert(alert.Sid)
-	// if err != nil {
-	// 	log.Println(err)
-	// 	w.WriteHeader(http.StatusInternalServerError)
-	// 	w.Header().Set("Content-Type", "text/plain")
-	// 	w.Write([]byte("failed alert lookup"))
-	// 	return
-	// }
+	splunk := splunk.SplunkServer(config.AppConfig.SplunkConfig)
```

Comment:
It's _probably_ fine, but could we either change the name of this var, or give an alias to the `"github.com/openshift/compliance-audit-router/pkg/splunk"` import? The variable name will shadow attempted usages of the package and could cause problems in the future.

Reply:
ACK, good call.  I'll rename it `splunkClient` and update the Jira "client" variable to `jiraClient`.  That's more accurate to the function, too.

Reply:
Actually, got rid of the variable entirely since it's only used immediately in the very next line.